src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -111,6 +111,13 @@ const getConnectionsAndInstallations = async (subscriptions: Subscription[], req
 	return { installations, successfulConnections, failedConnections };
 };
 
+const countStatus = (connections, syncStatus: string): number =>
+	connections && connections.filter(org => org?.syncStatus === syncStatus).length || 0;
+
```

Comment:
Can we add a type to `connections` for `countStatus` and `countNumberSkippedRepos` they look like `Subscription[]`?

Reply:
+1